src/layer.cpp

```diff
@@ -525,7 +525,7 @@ Mensur *Layer::GetCurrentMensur()
 const Mensur *Layer::GetCurrentMensur() const
 {
     const Staff *staff = vrv_cast<const Staff *>(this->GetFirstAncestor(STAFF));
-    assert(staff && staff->m_drawingStaffDef);
+    // assert(staff && staff->m_drawingStaffDef);
```

Comment:
I am curious about this... ;-) This is normally used in drawing code, so it is not available at the stage you are doing the scoring up. We can see if you can set the values before you call the scoring up, but how can this work when removing the assert?

Reply:
I don't exactly remember what happened. I get this message on the console when running the program:
```
Assertion failed: (staff && staff->m_drawingStaffDef), function GetCurrentMensur, file layer.cpp, line 529.
```
The message didn't go away and allow me to run things until I commented that line out. That was a long time ago, so I was still learning a bit about what I was doing. I would appreciate some feedback if you know what is causing this issue. Thanks!

Reply:
I will look once I have a test file

Reply:
Thank you! I just provided some test files below :)